Python/chapter01/1.1 - Is Unique/miguel_1.1_sol.py

```diff
@@ -0,0 +1,60 @@
+"""
+Python version 3.7.0
+1.1
+Implement an algorithm to determine if a string has all unique characters.
+What if you cannot use additional data structures?
+
+Given:		Expect:
+tacos		true
+swag		true
+bobby		false
+california	false
+orbit		true
+e		    true
+"""
+
+
+def is_unique(input_str):
+    freqs = {}
+    for c in input_str:
+        if c not in freqs:
+            freqs[c] = 1
+        else:
+            return False
+    return True
+
+
+def is_unique_no_additional_data_structures(input_str):
+    for i in range(0, len(input_str)):
```

Comment:
since you want both the index and the element, you can use [`enumerate()`](https://docs.python.org/3/library/functions.html#enumerate):

```suggestion
    for i, c in enumerate(input_str):
```